.env.example

```diff
@@ -8,3 +8,4 @@ SERVICE_PUSHER_BEAMS_INSTANCE_ID=
 SERVICE_PUSHER_BEAMS_SECRET_KEY=
 HUB_URL=
 SNAPSHOT_URI=
+REDIS_URL=
```

Comment:
Hi @wa0x6e It won't work without Redis? Redis is usually expensive, retry can't be in memory or on mysql DB?

Reply:
Most of the light queue libraries uses Redis as backend, it's built for that.

Redis is not used to store the retry, but is used as storage solution for the whole queue system.

As for the redis instance, only a small instance will be required, as data will be cleared as queue is cleared, only data for failed/waiting for retry jobs will be stored.
Another repo (envelop) is also using the queue system, we could share the same redis instance for all repo using a queue system

Reply:
For that we need @bonustrack 's approval 😢 because he denied Redis a few times as far as I remember

@bonustrack what you think? we could use Redis here?

Reply:
I see, I've seen `REDIS_URL` in a few places, so I just assumed that it was common.